src/server/server.ts

```diff
@@ -56,7 +56,9 @@ const prepareServer = ({
       // no-op
     }
 
-    await deno.runInBackground(['run', ...flags, stage2Path, port.toString()], true, processRef)
+    const bootstrapFlags = ['--port', port.toString()]
```

Comment:
looks like we didn't supply the `--port` flag before - assuming this is unrelated to the certificate path, was broken before, and you drive-by-fixed this.

Reply:
We did supply it, but as a positional parameter and not a flag (i.e. `deno run script.js <PORT>` as opposed to `deno run script.js --port <PORT>`). Using flags will allow us to supply additional parameters in the future in a more sustainable way.

You're right that it's not necessarily related to the certificate path, and I should've clarified that. Thanks for calling that out!

Reply:
See https://github.com/netlify/edge-functions-bootstrap/pull/30/files#r860951538.